cmd/argocd/commands/app.go

```diff
@@ -1272,12 +1268,25 @@ func waitOnApplicationStatus(acdClient apiclient.Client, appName string, timeout
 		if app.Operation != nil {
 			refresh = true
 		}
+
+		healthStatus := true
+		if watchSuspened {
+			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusSuspended
+		}
+
+		if watchHealth {
+			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusHealthy
+		}
+
+		if watchSuspened && watchHealth {
+			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusHealthy ||
+				app.Status.Health.Status == argoappv1.HealthStatusSuspended
+		}
```

Comment:
Can we use if/elses to make it more obvious that these are three separate conditions:

```golang
 		if watchSuspened && watchHealth {
			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusHealthy ||
				app.Status.Health.Status == argoappv1.HealthStatusSuspended
		} else if watchSuspened {
			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusSuspended
		} else if watchHealth {
			healthStatus = app.Status.Health.Status == argoappv1.HealthStatusHealthy
		}
```